packages/react-components/src/components/chart/eChartsConstants.ts

```diff
@@ -80,3 +80,5 @@ export const TREND_CURSOR_LINE_MARKERS_GRAPHIC_INDEX = 3;
 export const CHART_RESIZE_INITIAL_FACTOR = 0.75;
 export const CHART_RESIZE_MIN_FACTOR = 0.3;
 export const CHART_RESIZE_MAX_FACTOR = 0.75;
+
+export const MULTI_Y_AXIS_LEGEND_WIDTH = 172;
```

Comment:
curious : is this a constant , i.e do we want it to always occupy a fixed width ?